src/components/PageActions/PageActions.tsx

```diff
@@ -0,0 +1,37 @@
+import { getReportIssueURL } from "@site/src/utils/github-issue";
+import styles from "./PageActions.module.css";
+import Icon from "../Icon";
+import { useInkeepSearch } from "@site/src/hooks/useInkeepSearch";
+import BrowserOnly from "@docusaurus/BrowserOnly";
+
+const PageActions: React.FC<{ pathname: string }> = ({ pathname }) => {
+  const { setIsOpen, ModalSearchAndChat, inkeepModalProps } = useInkeepSearch({
+    enableAIChat: true,
+    chatOnly: true
+  });
+  return (
+    <div className={styles.pageActions}>
```

Comment:
I should have caught this sooner, but as I look at the implementation and see this across all pages, I think these actions are a bit too strong now. I've asked @rishibarbhaya-design to revisit in Figma.